### DISCUSSION TITLE
Stint on practice

### DISCUSSION THREAD

**Discussion by cofievet** [theOehrly/Fast-F1#394] [General]
Hi
For a start, thanks for the project and all this data. It's beneficial.
But when I look at practice data, I see something strange on the stint. 
The stint of the session starts with the last turn of previous

It's normal? 

Thanks and have a nice day

**Comment by theOehrly** [comment]
> The stint of the session starts with the last turn of previous

Hi, do you notice that with all sessions or only with some? Can you give me an example where that happens?

**Reply by cofievet** [reply]
I have notice that for all FP for the 2023 season (not check the other seasons for the moment) 

I think on qualifying it's the same thing

Thanks

**Reply by theOehrly** [reply]
@cofievet Can you please explain a bit more detailed what the problem is, in your opinion? I just had a look at some of the data and couldn't see anything that looks wrong.

**Reply by cofievet** [reply]
So I extract data into a database from your package for the round 7 of 2023 (Spain Grand Prix)
As you can see on the screen below, stint 2 for Verstappen, start at turn 5 with TEST_UNKNOW, then the MEDIUM type has used on turn 6
![image](https://github.com/theOehrly/Fast-F1/assets/15668319/f246d8c5-e34d-4541-8b5e-dad57b4e4f74)
The stint 2 needs to start on turn 6 with the compound MEDIUM

If you need more information don't hesitate

**Reply by theOehrly** [reply]
I don't have this problem in that session.

Are you maybe using an old version of FastF1? Please upgrade to the newest version and see if that fixes the problem.

**Reply by cofievet** [reply]
I have upgrade and the problem seems to be OK, 
Thanks and have a nice day
